war/src/main/less/modules/app-bar.less

```diff
@@ -0,0 +1,32 @@
+.jenkins-app-bar {
```

Comment:
What does app-bar mean? If it's for the configure screen, it should have a component-specific name. We already have CSS for the manage jenkins page.

Reply:
The app bar represents the heading for the page, it contains the title (in this case Manage Jenkins) with the option of placing components on the far right of it. I wanted to get it merged in with this MR as I plan on making extensive use of it in future design related MRs.